src/Avalonia.Diagnostics/Diagnostics/Views/ControlDetailsView.xaml

```diff
@@ -29,10 +29,38 @@
     
     <Grid Grid.Column="0" RowDefinitions="Auto,Auto,*">
 
-      <Grid ColumnDefinitions="Auto, *" RowDefinitions="Auto, Auto">
+      <Grid ColumnDefinitions="Auto, *,Auto" RowDefinitions="Auto, Auto">
         <Button Grid.Column="0" Grid.RowSpan="2" Content="^" Command="{Binding ApplyParentProperty}" />
         <TextBlock Grid.Column="1" Grid.Row="0" Text="{Binding SelectedEntityName}" FontWeight="Bold" />
         <TextBlock Grid.Column="1" Grid.Row="1" Text="{Binding SelectedEntityType}" FontStyle="Italic" />
+        <Button Command="{Binding $parent[local:MainView].DataContext.Shot}" Grid.Column="2" Grid.RowSpan="2">
```

Comment:
Do we need this button here?
I really don't think this functionality will be frequently used, and single menu action (top menu or context menu) is more than enough.  I also don't think we need to take more space in status bar, when hotkey can be displayed in the menu.